apps/server/src/services/TimerService.ts

```diff
@@ -94,33 +79,23 @@ export class TimerService {
   resume(timer: OntimeEvent, restorePoint: RestorePoint) {
     this._clear();
 
+    resumeTimer(timer, restorePoint);
+
     // this is pretty much the same as load, with a few exceptions
     this.loadedTimerId = timer.id;
     this.loadedTimerStart = timer.timeStart;
     this.loadedTimerEnd = timer.timeEnd;
 
-    this.timer.duration = calculateDuration(timer.timeStart, timer.timeEnd);
-    this.playback = restorePoint.playback;
-    this.timer.timerType = timer.timerType;
-    this.timer.endAction = timer.endAction;
-    this.timer.startedAt = restorePoint.startedAt;
-    this.timer.addedTime = restorePoint.addedTime;
     this.pausedTime = 0;
     this.pausedAt = restorePoint.pausedAt;
 
-    this.timer.current = this.timer.duration;
-    if (this.timer.timerType === TimerType.TimeToEnd) {
-      const now = clock.timeNow();
-      this.timer.current = getCurrent(now, this.timer.duration, 0, 0, now, timer.timeEnd, this.timer.timerType);
-    }
-
     this._onResume();
   }
```

Comment:
You can see what it will look like when this PR is ready. The service handles its state and executes an action that takes the server state. I'm still deciding if it's best to export `resumeTimer` or use another method.